src/lightning/pytorch/core/module.py

```diff
@@ -42,6 +42,7 @@
 from lightning.pytorch.core.optimizer import LightningOptimizer
 from lightning.pytorch.core.saving import ModelIO
 from lightning.pytorch.loggers import Logger
+from lightning.pytorch.trainer import call
```

Comment:
```suggestion
from lightning.pytorch.trainer import _calls
```
 seems all func there protected so let make whole module protected

Reply:
I'll leave this for a follow-up if we want to do it. We don't have any other protected modules and I don't want to stray away from the current pattern used in the trainer unless there's consensus